### PR TITLE
NTUEEPLUS-13. ProfilePage

### PR DESCRIPTION
**Pull request by featherchen**
### What is this PR for?

ProfilePage and Edit Rage
### What type of PR is it?
 Feature 

### Todos
* [ ] - hope not

### What is the Jira issue?

https://ntueeplus.atlassian.net/browse/NTUEEPLUS-13
### Screenshots (if appropriate)
Edit Page
Screenshot from 2021-07-22 23-43-58

Profile Page
Screenshot from 2021-07-22 21-14-54

### CODE REVIEW

**Comment by featherchen** [comment]
@noidname01

File: src/views/pages/profile/profile.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+body {
```
Comment (noidname01) [inline src/views/pages/profile/profile.css:1]: We don't need this file , right?

**Review by noidname01 — CHANGES_REQUESTED** [review]

File: src/views/pages/profile/profile.css (hunk 1 above)
Comment (featherchen) [reply src/views/pages/profile/profile.css]: @noidname01 done

**Comment by featherchen** [comment]
@noidname01

**Review by noidname01 — APPROVED** [review]
LGTM But I think we need to put svg to asset.

**Pull request merged**